components/builder-jobsrv/habitat/default.toml

```diff
@@ -16,8 +16,4 @@ connection_timeout_sec = 3600
 
 [archive]
 backend = "local"
-key = ""
-secret = ""
-endpoint = ""
-bucket = ""
-region = "us-east-1"
+local_dir = "/tmp"
```

Comment:
This is the default that we were really getting at from the change at our last customer off-site. We should only write to a service's directory. The other part of this commit was making `local_dir` automatically populate which is why this tunable was removed. You can see those changes [here](https://github.com/habitat-sh/habitat/blob/master/components/builder-jobsrv/habitat/config/config.toml#L18)